recipe/meta.yaml

```diff
@@ -12,10 +12,10 @@ source:
 
 build:
   number: 0
-  skip: true  # [py<=38 or win32 or s390x]
+  skip: true  # [py<38 or win32 or s390x or py>38 and not test_py39]
```

Comment:
I've added a variable `test_py39` to facilitate test builds of Python 3.9 and 3.10 support. Note this is never engaged in defaults builds, which means this will build only Python 3.8, as before.

Reply:
nit: `test_py39` includes py310 feels a bit weird lol. Maybe `test_min_py39` (or something that shows it is >= py39)?